app/components/agenda/agendaitem/decision/digital.js

```diff
@@ -312,12 +312,24 @@ export default class AgendaAgendaitemDecisionDigitalComponent extends Component
     const documents = this.pieces;
     const agendaActivity = await this.args.agendaitem.agendaActivity;
     const subcase = await agendaActivity?.subcase;
-    const newBetreftContent = await generateBetreft(shortTitle,
-      title,
-      this.args.agendaitem.isApproval,
-      documents,
-      subcase?.subcaseName
-    );
+    let newBetreftContent;
+    if (subcase?.isBekrachtiging) {
+      const ratification = await subcase.ratification;
+      newBetreftContent = await generateBetreft(
+        subcase.shortTitle,
+        null,
```

Comment:
We would probably still want to `agendaitem.shorttitle` here.
subcase title and agendaitem should be the same regardless